vscode/src/chat/chat-view/ChatPanelsManager.ts

```diff
@@ -312,12 +316,12 @@ export class ChatPanelsManager implements vscode.Disposable {
             this.disposeProvider(activePanelID)
         }
         // loop through the panel provider map
-        const panelsProvider = Array.from(this.panelProvidersMap.values())
+        const panelsProvider = Array.from(this.panelProviders.values())
```

Comment:
```suggestion
        const panelsProvider = Array.from(this.panelProviders)
```

do we still need Array.from if this.panelProviders is now an array?